gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/SessionTimeout.java

```diff
@@ -15,26 +15,31 @@ public class SessionTimeout implements Serializable {
 	private int refreshLimit = 0;
 	private String method = "";
 	private String additionalData = "";
+	private String signInUrl = "";
+	private SessionTimeoutTextOverrides textOverrides = null;
     
     public SessionTimeout() {
     }
 
     public SessionTimeout(int inactivity, int reactiontime, int sessionalive, 
     					  String logouturl, String refreshcallbackurl, 
     					  boolean refreshonclick, int refreshlimit, 
-    					  String method, String additionaldata) {
+    					  String method, String additionaldata,
+    					  String signinurl, SessionTimeoutTextOverrides textoverrides) {

```

Comment:
Since (I believe) this class is directly used by applications (right?), changing the constructor signature(s) will break people.  Basisically, the properties can be added to the constructor that takes everything, but a new one with its old signature must be created.  Likewise for this constructor, if it is desirable to have both the old signature AND one with the new properties, a new one will have to be created.  **If these properties are now REQUIRED by the WET control for the session timeout control to work, we'll have a chat** (bascially we'll have to decide on either proving default values or making a breaking release)